src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] According to the requirements all components should be class-based. Kindly refactor this component accordingly.

Reply:
You are right 

Thank you. Let me make the updates